youtube_dl/extractor/wdr.py

```diff
@@ -21,54 +21,41 @@ class WDRIE(InfoExtractor):
 
     _TESTS = [
         {
-            'url': 'http://www1.wdr.de/mediathek/video/sendungen/servicezeit/videoservicezeit560-videoplayer_size-L.html',
+            'url': 'http://www1.wdr.de/mediathek/video/sendungen/hier_und_heute/videostreetfoodpioniere100.html',
```

Comment:
I see the problem. The page I test only contains a single video and shouldn't return a playlist. I tried to separate the media extrator and playlist extractor to change that. Still in the same class because there doesn't seem to be any indicator in the url that the current page is a playlist.